test/PerformanceTests/Framework/TestService/Microsoft.Test.OData.TestService.csproj

```diff
@@ -13,12 +13,15 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.Test.OData.TestService</RootNamespace>
     <AssemblyName>Microsoft.Test.OData.TestService</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>

```

Comment:
shall we move to .NET Core

Reply:
Moving the TestService might require a lot of rewriting. I suggest we do that in a later PR once the current tests have been fixed.